linux-cachyos-pds/PKGBUILD

```diff
@@ -41,7 +41,7 @@ _makegconfig=${_makegconfig-}
 # NUMA is optimized for multi-socket motherboards.
 # A single multi-core CPU actually runs slower with NUMA enabled.
 # See, https://bugs.archlinux.org/task/31187
-_NUMAdisable=${_NUMAdisable-y}
+_NUMAdisable=${_NUMAdisable-}
```

Comment:
Is build with NUMA+PRJC really stable right now?

Reply:
Probably not